src/Sarif.Converters/Sarif.Converters.csproj

```diff
@@ -7,6 +7,10 @@
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
+  <PropertyGroup>
```

Comment:
It seems like about half the time you're letting build.props set the targets, and the other half you aren't - is it worth having build.props try to specify them at this point? It seems like it's not helping a lot, and then you add the gotcha that many projects need to ensure they set TargetFrameworks only after the import.

Reply:
Currently all product code requires it be sent after the import. For consistency, we should make the test assemblies work the same way.

---
In reply to: [391737075](https://github.com/microsoft/sarif-sdk/pull/1812#discussion_r391737075) [](ancestors = 391737075)